source/manual/how-to-be-a-good-buddy.html.md

```diff
@@ -0,0 +1,38 @@
+---
+owner_slack: "#govuk-developers"
+title: How to be a good buddy
+section: Learning GOV.UK
+layout: manual_layout
+type: learn
+parent: "/manual.html"
+---
+
+When a new joiner starts on govuk, it's good practice to assign them a "buddy" to help them get started on govuk and it's infrastructure and provide initial support.
+
+## What is a buddy?
+
+A buddy acts as a temporary guide for new joiners, helping them understand the govuk programme and generally get started. Some responsibilities of a buddy can include:
+
+- explaining how our stack works
+- pointing the new joiner to useful docs/slack channels/general resources
+- being on hand to answer specific questions
+- generally make them feel welcome
+
+Ideally, a buddy will be in the same team and sit within the same function as the new joiner eg: a new frontend developer would have a frontend buddy.
+
+## How does a buddy differ from a line manager?
+
+Line managers aren't the same as buddies in that they…
+
+- might not always be on the same team or even the same programme
+- are focused more on personal stuff like objectives and working patterns
+- have enough to deal with navigating SOP
+
+## Advice on being an effective buddy
+
+- Make sure that our new joiner documentation is up to date before they start
```

Comment:
Link to https://docs.publishing.service.gov.uk/manual/get-started.html?

(I did think perhaps to https://trello.com/b/yMWYSGlz/govuk-developer-onboarding instead, though this may still be awaiting sign-off from Senior Tech. And in any case, that feels more like a line manager's responsibility)